src/components/AllReviews.js

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from 'react';
+import { getReviews } from '../utils/api';
+import { Link } from 'react-router-dom';
+
+function AllReviews(){
+    const [reviews, setReviews] = useState([])
+    useEffect(() => {
+        getReviews().then((reviewsFromAPI) => {
+          console.log(reviewsFromAPI);
+          setReviews(reviewsFromAPI);
```

Comment:
Could do with a loading pattern so the user's not looking at a blank page while waiting for everything to load ⌛ 
If you're not sure how to do them there's an example at the end of these notes https://notes.northcoders.com/courses/js-front-end/useEffect